build/build-image/cross/Makefile

```diff
@@ -12,16 +12,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY:	build push
+.PHONY: all push container clean orphan all-push push-manifest
+
+include ../../../hack/make-rules/Makefile.manifest
+
+REGISTRY ?= staging-k8s.gcr.io
+IMAGE = $(REGISTRY)/kube-cross
+IMAGE_WITH_ARCH = $(IMAGE)-$(ARCH)
 
-IMAGE=kube-cross
 TAG=$(shell cat VERSION)
 
+# Architectures supported: amd64, arm64, ppc64le
+ARCH ?= amd64
+
+ALL_ARCH = amd64 arm64 ppc64le
+
+# If you want to build AND push all containers, see the 'all-push' rule.
+all: all-container
+
+all-push: all-push-images push-manifest
+
+push-manifest: manifest-tool
+	manifest-tool push from-args --platforms $(call join_platforms,$(ALL_ARCH)) --template $(IMAGE)-ARCH:$(TAG) --target $(IMAGE):$(TAG)
```

Comment:
I think we generally prefer using `docker manifest` now (e.g. the various `debian-*` images in `build/`)? though I guess this also works.

Reply:
Ack. let me switch this too